cluster/gce/trusty/configure.sh

```diff
@@ -261,7 +261,7 @@ health_monitoring() {
   # We simply kill the process when there is a failure. Another upstart job will automatically
   # restart the process.
   while [ 1 ]; do
-    if ! timeout 20 docker ps > /dev/null; then
+    if ! timeout 10 docker version > /dev/null; then
```

Comment:
I believe the `docker version` command will return immediately, with error code 1, even if the docker daemon is not running. Maybe `docker info` will provide a better test?

```
$ docker info
Cannot connect to the Docker daemon. Is the docker daemon running on this host?
```
```
$ echo $?
1
```

Also do we need to test the return code here?